backend/src/transaction/transaction.controller.ts

```diff
@@ -179,145 +178,89 @@ export default class TransactionController {
     }
   }
 
-  @Get(':publicKey')
+  @Get()
   @HttpCode(HttpStatus.OK) // 200 OK
   @ApiOkResponse({
     description: 'The transactions have been successfully retrieved.',
     type: [GetTransactionsResponseDto],
   })
   @ApiForbiddenResponse({ description: 'Forbidden', type: ForbiddenException })
-  @ApiParam({
-    name: 'publicKey',
-    description: 'The public key to retrieve transactions for',
-    example: 'cf8c984270cd7cd25e1bd6df1a3a22ee2d1cd53a0f7bbfdf917a8bd881b11b5e',
-    required: true,
-  })
-  @ApiQuery({
-    name: 'status',
-    description: 'The status of transaction to retrieve',
-    example: 'pending',
-    required: false,
-  })
-  @ApiQuery({
-    name: 'page',
-    description: 'The page number to retrieve',
-    example: 1,
-    required: false,
-  })
-  @ApiQuery({
-    name: 'limit',
-    description: 'The number of transactions to retrieve per page',
-    example: 10,
-    required: false,
-  })
+  @ApiQuery({ name: 'publicKey', description: 'The public key to retrieve transactions for', required: false })
```

Comment:
Why removing the examples? if we do that in swagger we will have to fill all fields all the time 🤔

Reply:
Agree, was an oversight